ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1048,6 +1048,244 @@ router bgp {{ router_bgp.as }}
+{%                 endif %}
+{%             endfor %}
+{%             for network in vrf.address_family_ipv4_multicast.networks | arista.avd.natural_sort('prefix') %}
+{%                 set network_cli = "network " ~ network.prefix %}
+{%                 if network.route_map is arista.avd.defined %}
+{%                     set network_cli = network_cli ~ " route-map " ~ network.route_map %}
+{%                 endif %}
+         {{ network_cli }}
+{%             endfor %}
+{%         endif %}
+{%         if vrf.address_family_ipv6 is arista.avd.defined %}
+      !
+      address-family ipv6
+{%             if vrf.address_family_ipv6.bgp.missing_policy.direction_in_action is arista.avd.defined %}
+         bgp missing-policy direction in action {{ vrf.address_family_ipv6.bgp.missing_policy.direction_in_action }}
+{%             endif %}
+{%             if vrf.address_family_ipv6.bgp.missing_policy.direction_out_action is arista.avd.defined %}
+         bgp missing-policy direction out action {{ vrf.address_family_ipv6.bgp.missing_policy.direction_out_action }}
+{%             endif %}
+{%             if vrf.address_family_ipv6.bgp.additional_paths.install is arista.avd.defined(true) %}
+         bgp additional-paths install
+{%             elif vrf.address_family_ipv6.bgp.additional_paths.install_ecmp_primary is arista.avd.defined(true) %}
+         bgp additional-paths install ecmp-primary
+{%             endif %}
+{%             if vrf.address_family_ipv6.bgp.additional_paths.receive is arista.avd.defined(true) %}
+         bgp additional-paths receive
+{%             endif %}
+{%             if vrf.address_family_ipv6.bgp.additional_paths.send.any is arista.avd.defined(true) %}
+         bgp additional-paths send any
+{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.backup is arista.avd.defined(true) %}
+         bgp additional-paths send backup
+{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.ecmp is arista.avd.defined(true) %}
+         bgp additional-paths send ecmp
+{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.ecmp_limit is arista.avd.defined %}
+         bgp additional-paths send ecmp limit {{ vrf.address_family_ipv6.bgp.additional_paths.send.ecmp_limit }}
+{%             elif vrf.address_family_ipv6.bgp.additional_paths.send.limit is arista.avd.defined %}
+         bgp additional-paths send limit {{ vrf.address_family_ipv6.bgp.additional_paths.send.limit }}
+{%             endif %}
+{%             for peer_group in vrf.address_family_ipv6.peer_groups | arista.avd.natural_sort('name') %}
+{%                 if peer_group.activate is arista.avd.defined(true) %}
+         neighbor {{ peer_group.name }} activate
+{%                 elif peer_group.activate is arista.avd.defined(false) %}
+         no neighbor {{ peer_group.name }} activate
+{%                 endif %}
+{%                 if peer_group.next_hop.address_family_ipv6_originate is arista.avd.defined(true) %}
```

Comment:
I don't find this in the schema.

Reply:
Copy-paste error, it's removed now. Thanks!